wls-common/src/main/java/org/jboss/arquillian/container/wls/CommonWebLogicConfiguration.java

```diff
@@ -527,4 +529,15 @@ public boolean isDeployExplodedArchive() {
    public void setDeployExplodedArchive(boolean deployExplodedArchive) {
        this.deployExplodedArchive = deployExplodedArchive;
    }
+
+   /**
+    * Remote machine configuration flag. Specify whether the target server is located on another machine (ie. if the
+    * target server do not share the same file system as the test server).
+    *
+    * @return {@code true} if executing test on a remote machine, {@code false} otherwise.
+    */
+   // Note: getter-only, setter must be implemented in child classes.
+   public boolean isRemoteMachine() {
+      return remoteMachine;

```

Comment:
Right now, I do not see what could possibly go wrong with using -remote also for local deployments (maybe deploying will takes a little while longer)... So yes, this seems a relatively safe assumption to make regarding the adminUrl.